src/components/Card.js

```diff
@@ -25,6 +43,25 @@ class Card {
     this._element = null;
   }
 
+  removeCard() {
+    this._element.remove();
+    this._lement = null;
+  }
+
+  updateLikeStatus(isLiked) {
+    this.isLiked = isLiked;
+    this._renderLikes();
+  }
+
+  _renderLikes() {
+    const likeButton = this._cardElement.querySelector(".card__like-button");
```

Comment:
You've found the likes button in line 75 as `this._likeButton`

No need to search it again